src/pages/special/edit/Edit.jsx

```diff
@@ -307,7 +307,7 @@ const Edit = () => {
                     </F.Text>
                   ) : null}
```

Comment:
popraw to na `{errors[FORMIK_HELPER.CREW] || touched[FORMIK_HELPER.CREW] && (
                    <F.Text className="validation-alert">
                      {errors[FORMIK_HELPER.CREW]}
                    </F.Text>)`